generated/nidcpower/nidcpower/session.py

```diff
@@ -326,12 +326,6 @@ class _SessionBase(object):
     power source to generate power. Use the power_source_in_use property to retrieve this information.
 
     Note: This property does not necessarily indicate if the device is using the auxiliary
-
```

Comment:
This deletion is not the result of my changes. Perforce changelist 49190802 caused this deletion.